tests/core_tests/config_tests.py

```diff
@@ -0,0 +1,129 @@
+
+#!/usr/bin/env python3
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2020 - 2023 Pionix GmbH and Contributors to EVerest
+
+from copy import deepcopy
+import logging
+from pathlib import Path
+import pytest
+import subprocess
+from tempfile import mkdtemp
+from typing import Dict
+
+from everest.testing.core_utils.fixtures import *
+from everest.testing.core_utils.everest_core import EverestCore
+
+from everest.testing.core_utils.configuration.everest_configuration_visitors.everest_configuration_visitor import \
+    EverestConfigAdjustmentVisitor
+
+
+@pytest.mark.everest_core_config('config-example.yaml')
+@pytest.mark.asyncio
+async def test_start_config_example(everest_core: EverestCore):
+    logging.info(">>>>>>>>> test_start_config_example <<<<<<<<<")
+    everest_core.start()
```

Comment:
Can you generalize this test and walk the directory for all `*.yaml` (except the one test which needs special handling)? That way every new config will automatically included, too. Besides removing a lot of boilerplate code.

Reply:
That was a surprisingly challenging thing to do with pytest, but I have a, arguably a bit hacky, solution implemented